scripts/observers/clean.py

```diff
@@ -0,0 +1,14 @@
+import shutil
+import os
+from pathlib import Path
+from dotenv import load_dotenv
+
+load_dotenv()
+
+working_dir = str(Path.home()) + str(os.getenv('WORKING_DIRECTORY'))
+
+try:
+    shutil.rmtree(working_dir)
+    print("done")
+except:
+    print('did nothing')
```

Comment:
`"nothing to clean" / "directory not removed"` or simply `pass`.

Reply:
refactored